src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -938,7 +939,7 @@ def _transform_subscription_for_multiapi(s, s_dict):
             s_dict[_MANAGED_BY_TENANTS] = [{_TENANT_ID: t.tenant_id} for t in s.managed_by_tenants]
 
 
-def _create_identity_instance(cli_ctx, *args, **kwargs):
+def _create_identity_instance(cli_ctx, authority, tenant_id=None, client_id=None):
```

Comment:
Very limited `args` (actually only one - `authority`) and `kwargs` are used. It's better to be explicit.

https://peps.python.org/pep-0020/

> Explicit is better than implicit.